README.md

```diff
@@ -1,33 +1,20 @@
-<!--
+[![CircleCI](https://circleci.com/gh/giantswarm/gitrepo.svg?style=shield)](https://circleci.com/gh/giantswarm/gitrepo)
+[![GoDoc](https://godoc.org/github.com/giantswarm/gitrepo?status.svg)](http://godoc.org/github.com/giantswarm/gitrepo)
```

Comment:
Nit: From the comment here:

> If the repository is public consider adding godoc badge. This should be
      the first badge separated with a single space